examples/contrib/tracking/tracking_svi_em.py

```diff
@@ -0,0 +1,286 @@
+from __future__ import absolute_import, division, print_function
```

Comment:
:+1: This looks much easier to collaborate on than Jupyter notebooks, thanks for figuring out how to do this!